pyinstaller-gui.py

```diff
@@ -76,7 +85,20 @@ def killapp(self, event):
         sys.exit(0)
 
     def makePackage(self, event):
-        commands = ['python3', 'pyinstaller.py']
+        if pyversion == 2:
+            commands = ['python', 'pyinstaller.py']
+
+        elif pyversion[0] == 3:
+            if sys.platform.startswith('win'):
+                if pyversion[1] > 2:  # python3.3+
+                    commands = ['py', '-3', 'pyinstaller.py']
+                else:
+                    commands = [sys.executable, 'pyinstaller.py']
+            else:
+                commands = ['python3', 'pyinstaller.py']
+        else:
```

Comment:
@matysek Should I wait with commiting until your commits are merged? Which commit precisely conflicts?
@htgoebel Maybe we could just use sys.executable for every case? It does work. But if 3.3+ becomes the minimum requirement "py -3" might be a better idea?